docs/css/main.css

```diff
@@ -509,17 +514,43 @@ th, td {
     width: auto;
     height: auto;
     padding: 16px 24px;
+    border-bottom: 1px solid #dadce0;
+  }
+
+  .toc > ul {
+    display: none;
+  }
+
+  .toc.open > ul {
+    display: block;
+  }
+
+  .toc > #toggleNavButton {
```

Comment:
class selector not necessary

Reply:
Mainly for readability - this section of the stylesheet is all about the toc.